src/components/Dashboard/Dashboard.module.scss

```diff
@@ -119,15 +126,65 @@
 .btn {
   width: 45%;
   padding: 0.5rem;
-  color: $severely-burnt-toast;
-  background-color: $translucent-silk;
-  border: none;
+  color: $translucent-silk;
+  background-color: $hornblende-green;
+  border: 1px solid $translucent-silk;
   border-radius: 10px;
   font-size: 1rem;
   font-weight: 300;
   cursor: pointer;
 }
 
 .btn:hover {
-  background-color: $spruce-stone;
+  background-color: $translucent-silk;
+  color: $severely-burnt-toast;
+}
+
+@media only screen and (min-width: 992px) {
```

Comment:
Create `breakpoint.scss` file where you will define breakpoints constants. The same that you did with the colors.


![CleanShot 2023-10-15 at 17 19 59](https://github.com/KamilaSemotanova/tracker/assets/12488648/d510ecb2-68a4-46fd-a0e0-5e6d4d61985c)

```
$screen-sm-min: 768px;
$screen-md-min: 992px;
$screen-lg-min: 1200px;
$screen-xl-min: 1440px;
```

Just a friendly recommendation. Define a few breakpoints and then never add new ones. Always go mobile first, that means min-width and NEVER mix min/max approach.

If you gonna break one of those rules above. Have extremely good reason for it :D, because what I came across in 99% of times, it was not needed